Library/Formula/cyclops.rb

```diff
@@ -0,0 +1,51 @@
+require "language/go"
+
+class Cyclops < Formula
+  desc "Docker-powered REPL for scripting and automation"
+  homepage "https://github.com/thisendout/cyclops"
+  url "https://github.com/thisendout/cyclops/archive/v0.1.0.tar.gz"
+  sha256 "93e636c67f644917410d8df04c5c46fb21e9ac69b9fe4ebbdfdda00010d0cdf8"
+
+  depends_on "go" => :build
+
+  cyclops_deps = %w[
+    github.com/mitchellh/gox a5a468f84d74eb51ece602cb113edeb37167912f
+    github.com/mitchellh/iochan 87b45ffd0e9581375c491fef3d32130bb15c5bd7
+  ]
+
+  cyclops_deps.each_slice(2) do |project, revision|
+    go_resource project do
+      url "https://#{project}.git", :revision => revision
+    end
+  end
```

Comment:
Since there are only two resources, isn’t it simpler to just write two `go_resource` instead of using a loop here?